.gitlab-ci.yml

```diff
@@ -18,7 +18,7 @@ stages:
 variables:
   # Format: $IMAGE-V$DATE [Cache is not used as of today but kept here
   # for reference]
-  CACHEKEY: "bionic_coq-V2019-12-08-V82"
+  CACHEKEY: "bionic_coq-V2020-02-28-V83"
```

Comment:
The CACHEKEY is incorrect [you are reusing an image, so you either bump of delete the old image in the interface]. Also, the bump is incorrect as it results in a non-buildable image due to dune problems.